lib/SILGen/SILGenDestructor.cpp

```diff
@@ -77,11 +77,12 @@ void SILGenFunction::emitDestroyingDestructor(DestructorDecl *dd) {
   }
 
 
-  if (cd->isDistributedActor()) {
-    SILBasicBlock *continueBB = createBasicBlock();
-    injectDistributedActorDestructorLifecycleCall(dd, selfValue, continueBB);
-    B.emitBlock(continueBB);
-  }
+//  // FIXME(distributed): enable this again; Assertion failed: (!hasValidInsertionPoint()), function emitBlock, file SILBuilder.h, line 334.
```

Comment:
not sure why it suddenly started failing... but I'm disabling everything for now just to get the initialization working first.